app/background/wallet/service.js

```diff
@@ -246,6 +246,15 @@ class WalletService {
     },
   );
 
+  getNonce = async (options) => {
+    this._ensureClient();
+    return this.client.getNonce(WALLET_ID, options.name, options);
+  };
+
+  importNonce = async (options) => {
+    return this._executeRPC('importnonce', [options.name, options.address, options.bid]);
```

Comment:
Does this require a wallet to be selected?

Reply:
This is handled in `_executeRPC()`:

https://github.com/kyokan/bob-wallet/blob/b365bbaf52d984c7a138e53514f50a645cf85d71/app/background/wallet/service.js#L311-L314

https://github.com/kyokan/bob-wallet/blob/b365bbaf52d984c7a138e53514f50a645cf85d71/app/background/wallet/service.js#L280-L294